src/client/backends/Msfs/Mapping.ts

```diff
@@ -98,7 +98,7 @@ export class MsfsMapping {
             if (runway.length > longestRunway[0]) {
                 longestRunway = [runway.length, runway];
             }
-            elevations.push(runway.elevation);
+            elevations.push(runway.elevation * 3.28084);
```

Comment:
Use the exact conversion instead.
```suggestion
            elevations.push(runway.elevation / 0.3048);
```